graphics/src/MeshManager.cc

```diff
@@ -822,6 +822,85 @@ void MeshManager::CreateCamera(const std::string &_name, float _scale)
   mesh->RecalculateNormals();
 }
 
+void MeshManager::CreateEllipsoid(const std::string &_name,
+                                  const ignition::math::Vector3d &_radii,
+                                  const unsigned int _rings,
+                                  const unsigned int _segments)
+{
+  if (this->HasMesh(_name))
```

Comment:
Can we add a test that tries to add the same mesh name, just for completeness :)